eodag/plugins/search/qssearch.py

```diff
@@ -1281,8 +1280,8 @@ def normalize_results(
         return products
 
     def discover_queryables(
-        self, product_type: Optional[str] = None
-    ) -> Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]:
```

Comment:
the return type is still Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]] ?